--- Makefile	2014-02-20 13:56:00.394683418 +0800
+++ Makefile	2014-02-20 13:57:19.990687241 +0800
@@ -18,6 +18,7 @@ PKG_MD5SUM:=79390673f5d07a8fb342bc09b5055b6f
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 
+include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
 
 define Package/libffi
@@ -71,4 +72,5 @@ define Package/libffi/install
 		$(1)/usr/lib/
 endef
 
+$(eval $(call HostBuild))
 $(eval $(call BuildPackage,libffi))
