diff -u --recursive glib-2.36.3-vanilla/configure.ac glib-2.36.3/configure.ac
--- glib-2.36.3-vanilla/configure.ac	2013-07-13 15:22:04.363505081 -0400
+++ glib-2.36.3/configure.ac	2013-07-13 15:25:42.463804445 -0400
@@ -3710,19 +3710,6 @@
 gio/tests/gdbus-object-manager-example/Makefile
 gio/tests/services/Makefile
 gio/tests/services/org.gtk.GDBus.Examples.ObjectManager.service
-po/Makefile.in
-docs/Makefile
-docs/reference/Makefile
-docs/reference/glib/Makefile
-docs/reference/glib/version.xml
-docs/reference/gobject/Makefile
-docs/reference/gobject/version.xml
-docs/reference/gio/Makefile
-docs/reference/gio/gdbus-object-manager-example/Makefile
-docs/reference/gio/version.xml
-tests/Makefile
-tests/gobject/Makefile
-tests/refcount/Makefile
 m4macros/Makefile
 ])
 
diff -u --recursive glib-2.36.3-vanilla/glib/Makefile.am glib-2.36.3/glib/Makefile.am
--- glib-2.36.3-vanilla/glib/Makefile.am	2013-07-13 15:22:04.393503884 -0400
+++ glib-2.36.3/glib/Makefile.am	2013-07-13 15:26:29.260938004 -0400
@@ -36,10 +36,7 @@
 endif
 
 SUBDIRS = libcharset $(PRINTF_SUBDIR) $(MAYBE_PCRE) update-pcre .
-DIST_SUBDIRS = libcharset gnulib pcre update-pcre tests
-if BUILD_MODULAR_TESTS
-SUBDIRS += tests 
-endif
+DIST_SUBDIRS = libcharset gnulib pcre update-pcre
 
 AM_CPPFLAGS = 				\
 	$(glib_INCLUDES) 		\
diff -u --recursive glib-2.36.3-vanilla/gobject/Makefile.am glib-2.36.3/gobject/Makefile.am
--- glib-2.36.3-vanilla/gobject/Makefile.am	2013-07-13 15:22:04.269508832 -0400
+++ glib-2.36.3/gobject/Makefile.am	2013-07-13 15:26:40.204501548 -0400
@@ -5,9 +5,6 @@
 include $(top_srcdir)/Makefile.decl
 
 SUBDIRS = .
-if BUILD_MODULAR_TESTS
-SUBDIRS += tests 
-endif
 
 BUILT_SOURCES=
 CLEANFILES=
diff -u --recursive glib-2.36.3-vanilla/Makefile.am glib-2.36.3/Makefile.am
--- glib-2.36.3-vanilla/Makefile.am	2013-07-13 15:22:04.361505160 -0400
+++ glib-2.36.3/Makefile.am	2013-07-13 15:27:15.307101667 -0400
@@ -6,13 +6,8 @@
 
 ACLOCAL_AMFLAGS = -I m4macros ${ACLOCAL_FLAGS}
 
-SUBDIRS = . m4macros glib gmodule gthread gobject gio po docs
+SUBDIRS = . m4macros glib gmodule gthread gobject gio
 DIST_SUBDIRS = $(SUBDIRS) build
-if BUILD_MODULAR_TESTS
-SUBDIRS += tests 
-else
-DIST_SUBDIRS += tests
-endif
 
 bin_SCRIPTS = glib-gettextize
 
